### ISSUE TITLE
Implement a queue/playlist

### ISSUE THREAD

**Issue by wrigglebug** [ysdragon/StreamBot#39]
It would be nice to be able to queue multiple files to be streamed

**Comment by ysdragon** [comment]
Yes, i have already implemented this feature in the rewritten version that i will release when i have some time.

**Comment by Marco5dev** [comment]
![Image](https://github.com/user-attachments/assets/f5e6de31-57a2-4e37-a4ef-71324fcede4a)
The developer is rude

**Comment by FurtMyGurt** [comment]
I was wondering how I set up a playlist? I saw that you added it into your rewritten version but I don't see any command prompts that would inhere

**Comment by ysdragon** [comment]
This has been implemented in this commit (https://github.com/ysdragon/StreamBot/commit/b6d48e8aef3de130245aab0998b5bdd18839c2a2)
